### PR TITLE
Update browser compatibility doc for background blur

### PR DESCRIPTION
**Pull request by davcolle**
**Issue #:**

**Description of changes:**

Updating background blur docs to indicate that iOS is not supported.

**Testing:**

* Manual tested in browser app in Safari, Chrome, and Firefox
* Added unit tests check iOS user agents return false in blur processor isSupported method

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*

Background blur is integrated into the browser demo application. To try it out, launch the demo with npm run start, join the meeting, click on the camera icon to enable video, then select the filter drop down and select Background Blur.

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
Yes

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
No

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: guides/15_Background_Filter_Video_Processor.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Background blur is available as part of the Amazon Chime SDK for JavaScript from
 
 ### Browser compatibility
 
-Background blur in the Amazon Chime SDK for JavaScript works in Firefox, Chrome, and Chromium-based browsers (including Electron) on desktop, Android, and iOS operating systems, and Safari 14.1 and later on macOS and some iOS devices. A full compatibility table is below.
+Background blur in the Amazon Chime SDK for JavaScript works in Firefox, Chrome, and Chromium-based browsers (including Electron) on desktop, Android, and Safari 14.1 and later on macOS.
```
Comment (michhyun1) [inline guides/15_Background_Filter_Video_Processor.md:19]: can we explicitly mention its not working on safari 15?

File: src/backgroundblurprocessor/BackgroundBlurVideoFrameProcessor.ts [hunk 2]
```diff
@@ -221,13 +221,32 @@ export default class BackgroundBlurVideoFrameProcessor {
       return Promise.resolve(false);
     }
 
+    const isIOS =
+      this.isIOSChrome(browser) || this.isIOSFirefox(browser) || this.isIOSSafari(browser);
```
[the remark this answers is not in the document]
Comment (michhyun1) [reply src/backgroundblurprocessor/BackgroundBlurVideoFrameProcessor.ts:225]: If you can leave a comment with the webkit above the function thatd be great too

[the remark this answers is not in the document]
Comment (ltrung) [reply src/backgroundblurprocessor/BackgroundBlurVideoFrameProcessor.ts:225]: We should just group all of this (including the safari check above) in a browserBehavior function called `isBackgroundBlurSupported`

**Review by ltrung — COMMENTED** [review]
Need CHANGELOG update.

**Pull request merged**